setup.py

```diff
@@ -10,7 +10,7 @@
     packages=find_packages(),
     zip_safe=False,
     install_requires=[
-        'django>=1.7a,<1.9',
+        'django>=1.10,<1.11',
```

Comment:
trying https://docs.travis-ci.com/user/languages/python/#Testing-Against-Multiple-Versions-of-Dependencies-(e.g.-Django-or-Flask)